src/components/PokemonList/SearchBar.tsx

```diff
@@ -0,0 +1,60 @@
+import {
+  ActivityIndicator,
+  TextInput,
+  View,
+  Text,
+  StyleSheet,
+} from "react-native";
+import Animated from "react-native-reanimated";
+
+type Props = {
+  animatedStyle: { top: number };
```

Comment:
I don't think it is proper type for `animatedStyle` 🤔

Reply:
![Screenshot 2024-09-04 at 14 26 38](https://github.com/user-attachments/assets/574faee6-1bdd-42d4-a1d6-ee9ac53a85eb)

Assumed that in my case is enough to allow animated only change of top

Reply:
In that case it should not be called `animatedStyle` but rather something like: `topPositionStyle` (or better).
`animatedStyle` suggest that this is style property and any style could be passed into it.

Reply:
I think it would make more sense to use sth like:
```suggestion
  animatedStyle: StyleProp<ViewStyle>;
```